.github/workflows/run-tests.yaml

```diff
@@ -96,21 +96,21 @@ jobs:
           PG_LOGGER: debug
           REDIS_URL: redis://localhost:${{ job.services.redis.ports[6379] }}
           MQ_REDIS_URL: redis://localhost:${{ job.services.redis.ports[6379] }}
-  build-docker-images:
```

Comment:
This one isnt needed anymore?

Reply:
I was testing it just now and reverted the changes